### PR TITLE
feat(browser-starfish): fetch real resources in resource module

### PR DESCRIPTION
**Pull request by DominikB2014**
3 main changes.
1. Fetch actual resource data (exception here is the size, uncompressed, and render blocking, we're working on adding those)
2. Allow sorting on avg duration
3. Allow filtering on javascript or image resoruces in the resource type dropdown.

image

### CODE REVIEW

File: static/app/views/performance/browser/resources/index.tsx [hunk 1]
```diff
@@ -116,9 +116,9 @@ function ResourceTypeSelector({value}: {value?: string}) {
 
   const options: Option[] = [
     {value: '', label: 'All'},
-    {value: '.js', label: 'Javscript (.js)'},
+    {value: 'resource.script', label: 'Javscript (.js)'},
```
Comment (AbhiPrasad) [inline static/app/views/performance/browser/resources/index.tsx:119]: do we need to translate the labels?

```suggestion
    {value: 'resource.script', label: 'JavaScript (.js)'},
```

File: static/app/views/performance/browser/resources/resourceTable.tsx [hunk 2]
```diff
@@ -90,12 +93,19 @@ function ResourceTable({sort}: Props) {
...
+    }
     if (key === 'http.response_content_length') {
       return <FileSize bytes={row[key]} />;
     }
-    if (key === 'avg(span.duration)') {
+    if (key === 'avg(span.self_time)') {
       return <DurationCell milliseconds={row[key]} />;
     }
+    if (key === 'span.op') {
+      const opName = row[key] === 'resource.script' ? 'Javascript' : 'Image';
```
Comment (AbhiPrasad) [inline static/app/views/performance/browser/resources/resourceTable.tsx:106]: do we need to translate these?

**Review by AbhiPrasad — APPROVED** [review]
We should add starfish as codeowners to these files

File: static/app/views/performance/browser/resources/resourceTable.tsx (hunk 2 above)
Comment (DominikB2014) [reply static/app/views/performance/browser/resources/resourceTable.tsx:106]: Yeah we probably should, good catch!

**Pull request merged**